h2o-r/tests/testdir_munging/unop/runit_mean_axis.R

```diff
@@ -0,0 +1,79 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+##
+# Test out the h2o.mean() functionality
+##
+
+
+test.mean <- function(){
+  #Set up H2OFrame
+  fr = h2o.createFrame(rows = 100, cols = 10, categorical_fraction = 0,
+                    factors = 0, integer_fraction = 1.0, integer_range = 100,
+                    binary_fraction = 0, time_fraction = 0, string_fraction = 0,
+                    has_response = FALSE, missing_fraction = 0.10)
+
+  ################################################################
+  #Testing h2o.mean and have a frame returned
+  mean_return_frame = h2o.mean(fr,return_frame = TRUE)
+
+  #Testing h2o.mean and have a list returned
+  mean_return_list = h2o.mean(fr)
+
+  #Check types returned are correct
+  expect_equal(class(mean_return_frame),"H2OFrame")
+  expect_equal(class(mean_return_list),"numeric")
+
+  ################################################################
+  #Check axis arguments set to 1 (row mean) and compare to R
+  mean_frame_axis = h2o.mean(fr,na.rm=TRUE,axis=1,return_frame=TRUE)
+  mean_list_axis = mean(t(fr),na.rm=TRUE)
```

Comment:
why list axis uses `mean` and frame axis `h2o.mean`? here and later in the script